packages/Builtins/_.candy

```diff
@@ -37,22 +37,51 @@ isLessThanOrEqualTo a b =
     Greater -> False
 
 equals a b :=
-  # Returns `True` or `False` depending on whether `a` and `b` are equal.
+  # Returns `True` if both values are equal, otherwise `False`.
+  #
+  # TODO: Document how equality works. Also, change how equality for
+  # functions works.
+  #
+  # ```
+  # equals 5 5 => True
+  # equals 5 2 => False
+  # ```
   ✨.equals a b
 
 functionRun function :=
-  # Executes `function` and returns its result.
+  # Calls the `function` with zero arguments. Returns the return value of the
+  # function.
+  #
+  # This built-in is necessary because functions in Candy are called by writing
+  # space-separated arguments such as `foo 2`. For functions that take no
+  # arguments, this syntax is not possible. Such functions are rare though
+  # because most functions don't have side effects.
   #
-  # `function` may not accept any arguments.
+  # ```
+  # functionRun { 4 } => 4
+  # ```
   needs (function | typeIs Function)
   needs (function | ✨.getArgumentCount | equals 0)
   ✨.functionRun function
+
 getArgumentCount function :=
-  # Returns the number of arguments that `function` expects.
+  # Returns the number of arguments the `function` requires.
+  #
+  # ```
+  # getArgumentCount { a -> a } => 1
+  # getArgumentCount getArgumentCount => 1
+  # ```
   needs (function | typeIs Function)
   ✨.getArgumentCount function
 
 ifElse condition then else :=
+  # If the condition is `True`, runs `then`. If it's `False`, runs `else`.
```

Comment:
```suggestion
  # If the condition evaluates to `True`, runs `then`. If it's `False`, runs `else`.
```

Reply:
From the view of the `ifElse` function, the `condition` parameter should just _be_ `True` – the expression is evaluated before the call even happens. So I'd actually leave it as-is.